src/app/home/avatar-generator/AvatarUtils.ts

```diff
@@ -0,0 +1,87 @@
+import {AvatarOptions, KeyAvatar} from './Avatartypes';
+import { getPreview } from './AvatarSelectorPath';
+
```

Comment:
Please update the name of this file to use kebab-case, to keep it consistent with the rest of the files.